Fix nightly 2025-03-03

SSH Timeout

https://github.com/gardenlinux/gardenlinux/actions/runs/13624454199/job/38083723322

https://github.com/gardenlinux/gardenlinux/actions/runs/13624454199/job/38082213824

```
module.aws["aws-gardener_prod_trustedboot_tpm2-amd64"].null_resource.test_connect (remote-exec):   Private key: true
module.aws["aws-gardener_prod_trustedboot_tpm2-amd64"].null_resource.test_connect (remote-exec):   Certificate: false
module.aws["aws-gardener_prod_trustedboot_tpm2-amd64"].null_resource.test_connect (remote-exec):   SSH Agent: false
module.aws["aws-gardener_prod_trustedboot_tpm2-amd64"].null_resource.test_connect (remote-exec):   Checking Host Key: false
module.aws["aws-gardener_prod_trustedboot_tpm2-amd64"].null_resource.test_connect (remote-exec):   Target Platform: unix
module.aws["aws-gardener_prod_trustedboot_tpm2-amd64"].null_resource.test_connect: Still creating... [9m30s elapsed]
module.aws["aws-gardener_prod_trustedboot_tpm2-amd64"].null_resource.test_connect: Still creating... [9m40s elapsed]
module.aws["aws-gardener_prod_trustedboot_tpm2-amd64"].null_resource.test_connect (remote-exec): Connecting to remote host via SSH...
module.aws["aws-gardener_prod_trustedboot_tpm2-amd64"].null_resource.test_connect (remote-exec):   Host: 35.156.203.55
module.aws["aws-gardener_prod_trustedboot_tpm2-amd64"].null_resource.test_connect (remote-exec):   User: admin
module.aws["aws-gardener_prod_trustedboot_tpm2-amd64"].null_resource.test_connect (remote-exec):   Password: false
module.aws["aws-gardener_prod_trustedboot_tpm2-amd64"].null_resource.test_connect (remote-exec):   Private key: true
module.aws["aws-gardener_prod_trustedboot_tpm2-amd64"].null_resource.test_connect (remote-exec):   Certificate: false
module.aws["aws-gardener_prod_trustedboot_tpm2-amd64"].null_resource.test_connect (remote-exec):   SSH Agent: false
module.aws["aws-gardener_prod_trustedboot_tpm2-amd64"].null_resource.test_connect (remote-exec):   Checking Host Key: false
module.aws["aws-gardener_prod_trustedboot_tpm2-amd64"].null_resource.test_connect (remote-exec):   Target Platform: unix
module.aws["aws-gardener_prod_trustedboot_tpm2-amd64"].null_resource.test_connect: Still creating... [9m50s elapsed]
╷
│ Error: remote-exec provisioner error
│ 
│   with module.aws["aws-gardener_prod_trustedboot_tpm2-amd64"].null_resource.test_connect,
│   on modules/aws/main.tf line 302, in resource "null_resource" "test_connect":
│  302:   provisioner "remote-exec" {
│ 
│ timeout - last error: dial tcp 35.156.203.55:22: connect: connection refused
╵
make: *** [Makefile:64: aws-gardener_prod_trustedboot_tpm2-amd64-apply] Error 1
make: Leaving directory '/home/runner/work/gardenlinux/gardenlinux/tests/platformSetup'
```

--- a/features/_usi/image.esp.tar
+++ b/features/_usi/image.esp.tar
@@ -61,7 +61,7 @@ chroot "$rootfs" dracut \
 	--force \
 	--tmpdir /tmp \
 	--kver "${kernel#*-}" \
-	--modules "bash dash systemd systemd-initrd systemd-repart kernel-modules kernel-modules-extra terminfo udev-rules dracut-systemd base fs-lib shutdown crypt" \
+	--modules "bash dash systemd systemd-initrd systemd-repart systemd-cryptsetup kernel-modules kernel-modules-extra terminfo udev-rules dracut-systemd base fs-lib shutdown crypt" \
 	--reproducible \
 	--install 'find findmnt grep lsblk tail head realpath basename sfdisk mkfs.ext4 resizefat32 dd cryptsetup' \
 	--include '/tmp/initrd.include' / \

